trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py

```diff
@@ -0,0 +1,195 @@
+## @ingroup Methods-Geometry-Two_Dimensional-Cross_Section-Airfoil
+# compute_airfoil_polars.py
+# 
+# Created:  Mar 2019, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+from SUAVE.Core               import Data , Units
+from .import_airfoil_geometry import import_airfoil_geometry 
+from .import_airfoil_polars   import import_airfoil_polars
+import numpy as np
+
+## @ingroup Methods-Geometry-Two_Dimensional-Cross_Section-Airfoil
+def compute_airfoil_polars(propeller,airfoils):
+    """This omputes models of Lift and Drag Coefficients of Stalled and Unstalled Airfoils.
```

Comment:
Can you expand this docstring? It's not very clear to me what it's doing.

Reply:
Docstring updated with more detailed description